scripts/fires/test_data/expected.csv

```diff
@@ -0,0 +1,6 @@
+dcid,name,typeOf,Location,FireCause,FireCauseGeneral,FireCauseSpecific,FireDiscoveryDateTime,ControlDateTime,ContainmentDateTime,BurnedArea,Costs,TotalIncidentPersonnel,IrwinID,wfigsFireID,ParentFire,InitialResponseDateTime,InitialResponseAcres
+fire/IRWINID-{7026435a-b39a-4cb1-bc71-a5b31714cb45},Chicken Coop Woodland Fire,PrescribedFireEvent,"dcid:geoId/13, dcid:geoId/13213, dcid:country/USA, [LatLong 34.9272 -84.6506]",FireCauseUnknown,,,2022-04-28 09:50:59,,,[Acre 1.0],,,{7026435a-b39a-4cb1-bc71-a5b31714cb45},2022-GACHF-220135,,,[Acre 74.0]
```

Comment:
it's unusual to have brackets in the dcid. in this case, it seems we can skip this. how about:
dcid: IrwinId/7026435a-b39a-4cb1-bc71-a5b31714cb45

/cc @pradh

Reply:
Done